From 6c922f988c9bc167ab2929e75fe482a9a5bca092 Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz.fischer@ettus.com>
Date: Mon, 14 Dec 2015 15:39:49 -0800
Subject: [PATCH] zynq: e3xx: Set factory dtb, otherwise it doesn't boot.

This fixes an issue where if the MB EEPROM is not initialized,
device_tree_image env would not get set, and it would not boot.

Signed-off-by: Moritz Fischer <moritz.fischer@ettus.com>
---
 board/ettus/e3xx/board.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/ettus/e3xx/board.c b/board/ettus/e3xx/board.c
index e056209..b8739c9 100644
--- a/board/ettus/e3xx/board.c
+++ b/board/ettus/e3xx/board.c
@@ -209,6 +209,7 @@ static void identify_products(void)
 	} else {
 		speedgrade = 0;
 		printf("*** Found unknown motherboard, please update sd card ***\n");
+		setenv("devicetree_image", "uImage-zynq-e3xx-factory.dtb");
 	}
 
 	/* print out daughterboard info and select correct image */
-- 
2.4.3

